src/gc/env/gcenv.os.h

```diff
@@ -277,6 +277,12 @@ class GCToOSInterface
     // Return:
     //  Time stamp in milliseconds
     static uint32_t GetLowPrecisionTimeStamp();
+
+    // Gets the current instruction pointer.
+    // Return:
+    //  The value of the instruction pointer immediately after
+    //  the call to this function.
+    static void* GetCurrentInstructionPointer();
```

Comment:
I do not think this should be on GCToOSInterface. Getting current IP is not OS service, it is more like compiler intrinsic service. If we really wanted to have it, it should be together with other compiler intrinsics, e.g. there is a cluster of them in gcenv.base.h - look for MemoryBarrier.

However, I am not sure whether this is needed at all: There are just two odd places (more like bugs than features) where this is called, and they are distinguished by the error code already, the IP is not necessary to distinguish them. We should just have the error code in the callback, and not worry about the IP. It should be fine to call `EEPOLICY_HANDLE_FATAL_ERROR` on the VM side whatever IP it happens to pick up.